### PR TITLE
STAC-0 Fix prediction script API mistakes

### PR DESCRIPTION
**Pull request by lowdewijk**

### CODE REVIEW

File: develop/scripting/script-apis/prediction.md [hunk 1]
```diff
@@ -8,28 +8,33 @@ description: Functions for predicting data available in StackState.
 
 ## Function: `predictMetrics`
 
-The function executes asynchronously
+Predict metrics for the 
 
 **Args:**
 
 * `predictorName` - name of prediction preset. Current available predictors: `linear` and `hmn`.
```
Comment (konstantintatarnikov) [inline develop/scripting/script-apis/prediction.md:15]: we also have `fft` - predictor

File: develop/scripting/script-apis/prediction.md [hunk 2]
```diff
@@ -8,28 +8,33 @@ description: Functions for predicting data available in StackState.
...
-* `horizon` - how much future to predict. The horizon is specified in the REF format.
-* `query` - what telemetry to predict. The query is specified in the STQL format REF.
+* `horizon` - how much future to predict. The horizon is specified in the [duration format](scripting/script-apis/time.md).
+* `query` - what metrics to use for the prediction. The query is specified in the STQL format.
+
+**Builder methods:**
+ 
+ * `predictionPoints(points: Int)` - the number of points to the horizon.
+ * `includeHistory(start?: Instant, end?: Instant)` - call this builder method to include the result of the `query` in the return value. Optionally a start and end can be added to limit the included history using the [instant format](scripting/script-apis/time.md).
```
Comment (konstantintatarnikov) [inline develop/scripting/script-apis/prediction.md:22]: it is either `includeHistory(start)` or `includeHistory(start, end)` or `includeHistory()` (=> for the last one it is not clear what's going to happen, probably all history is taken)

File: develop/scripting/script-apis/prediction.md [hunk 3]
```diff
@@ -8,28 +8,33 @@ description: Functions for predicting data available in StackState.
...
-The function executes asynchronously
+Predict metrics for the 
 
 **Args:**
 
 * `predictorName` - name of prediction preset. Current available predictors: `linear` and `hmn`.
-* `horizon` - how much future to predict. The horizon is specified in the REF format.
-* `query` - what telemetry to predict. The query is specified in the STQL format REF.
+* `horizon` - how much future to predict. The horizon is specified in the [duration format](scripting/script-apis/time.md).
+* `query` - what metrics to use for the prediction. The query is specified in the STQL format.
```
Comment (konstantintatarnikov) [inline develop/scripting/script-apis/prediction.md:17]: the query expected is actually TelemetryMetricQuery from the code

File: develop/scripting/script-apis/prediction.md (hunk 2 above)
Comment (lowdewijk) [reply develop/scripting/script-apis/prediction.md]: Clarified it.

File: develop/scripting/script-apis/prediction.md (hunk 3 above)
Comment (lowdewijk) [reply develop/scripting/script-apis/prediction.md]: added another clarification

**Pull request merged**